.github/workflows/mlkem.yml

```
@@ -168,3 +168,35 @@ jobs:
         run: |
           cargo clean
           cargo hack test --each-feature $EXCLUDE_FEATURES --verbose $RUST_TARGET_FLAG
+
+  fuzz:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - macos-latest # macos-14 m1
+          - ubuntu-latest
+
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+        working-directory: libcrux-ml-kem
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: 🛠️ Setup Rust Nightly
+        run: rustup toolchain install nightly
+
+      - name: Update dependencies
+        run: cargo update
+
+      - run: 🏃🏻‍♀️ Decaps
```

```suggestion
      - name: 🏃🏻‍♀️ Decaps
```
Ditto the places where `run` is duplicated below.